code/defines/procs/dbcore.dm

```diff
@@ -128,7 +128,7 @@ DBQuery/proc/Execute(var/list/argument_list = null, var/pass_not_found = 0, sql_
 
 	var/error = ErrorMsg()
 	if (error)
-		error("SQL Error: '[error]'")
+		log_debug("SQL Error: '[error]'",SEVERITY_ERROR)

```

Comment:
IIRC mods have access to debug logs. How high are the chances of something sensitive (connection details) ending up in this? I forgetti.